docs/admin_guide.md

```diff
@@ -99,15 +99,13 @@ If the result is empty, make sure [`pg_diffix` is loaded](#using-the-extension).
 
 ### Anonymization salt
 
-During the anonymization process, random noise needs to be generated at various stages. The noise has to be unpredictable,
-but also consistent for the same dataset. Salting the noise generator is the mechanism that ensures this happens.
+The operation of Diffix Elm requires a per-database secret salt value. The salt can only be viewed or set by superusers, and __must be kept secret__.
 
-The salt is a secret value that is consistent for a certain dataset. __Warning__: If the same dataset is exposed from multiple
-systems, the salt has to be synchronized between the different systems to prevent information leakage.
+`pg_diffix` automatically generates a salt value when a database is added.
```

Comment:
What does it mean that "a database is added"? The initialization script runs when the extension is manually activated for a database (by executing the command `CREATE EXTENSION pg_diffix;`), not when a database is created.